concepts/recursion/about.md

```diff
@@ -0,0 +1,176 @@
+# About
+
+Recursion is a way to repeatedly execute code inside a function through the function calling itself.
+Functions that call themselves are know as _recursive_ functions.
+Recursion can be viewed as another way to loop/iterate. 
+And like looping, a Boolean expression or `True/False` test is used to know when to stop the recursive execution.
+_Unlike_ looping, recursion without termination in Python cannot not run infinitely.
+Values used in each function call are placed in their own frame on the Python interpreter stack.
+If the total amount of function calls takes up more space than the stack has room for, it will result in an error.
+
```

Comment:
_Here is where I would insert a write-up on "#7" -- how looping and recursion "feel" like they execute the same way, but they look very different when you compare steps, calls, and how each uses stack frames._